example-apps/kitchen-sink-example/source/components/form-controls/my-form.ts

```diff
@@ -30,7 +30,7 @@ import {NgForm} from '@angular/common';
   directives: [FORM_DIRECTIVES]
 })
 export default class MyForm {
-  onSubmit(regForm: NgForm) {
-    console.log(regForm.value);
-  }
+  // onSubmit(regForm: NgForm) {
```

Comment:
@sumitarora It seems like it's not done? I'm looking at the "Files changed" tab and it seems it's still commented instead of removed.